project-base/app/src/DataFixtures/Demo/ComplaintDataFixture.php

```diff
@@ -54,15 +31,15 @@ public function load(ObjectManager $manager): void
         $customerUser1 = $this->getReference(CustomerUserDataFixture::CUSTOMER_PREFIX . 1);
         /** @var \App\Model\Order\Order $order1 */
         $order1 = $this->getReference(OrderDataFixture::ORDER_PREFIX . 1);
-        $uploadedFile1 = $this->createUploadedFiles(__DIR__ . '/../resources/images/complaint/400.jpg');
-        $uploadedFile2 = $this->createUploadedFiles(__DIR__ . '/../resources/images/complaint/401.jpg');
-        $uploadedFile3 = $this->createUploadedFiles(__DIR__ . '/../resources/images/complaint/402.jpg');
-        $uploadedFile4 = $this->createUploadedFiles(__DIR__ . '/../resources/images/complaint/403.jpg');
+        $uploadedFile1 = $this->complaintHelper->createUploadedFiles(__DIR__ . '/../resources/images/complaint/400.jpg');
+        $uploadedFile2 = $this->complaintHelper->createUploadedFiles(__DIR__ . '/../resources/images/complaint/401.jpg');
+        $uploadedFile3 = $this->complaintHelper->createUploadedFiles(__DIR__ . '/../resources/images/complaint/402.jpg');
+        $uploadedFile4 = $this->complaintHelper->createUploadedFiles(__DIR__ . '/../resources/images/complaint/403.jpg');
 
         $orderItems1 = $order1->getProductItems();
-        $orderItem1 = $this->createComplaintItemData(array_shift($orderItems1), 'Both broken!', 2, [$uploadedFile1, $uploadedFile2]);
-        $orderItem2 = $this->createComplaintItemData(array_shift($orderItems1), 'Broken!', 1, [$uploadedFile3]);
-        $complaint1 = $this->createComplaint(
+        $orderItem1 = $this->complaintHelper->createComplaintItemData(array_shift($orderItems1), 'Both broken!', 2, [$uploadedFile1, $uploadedFile2]);
+        $orderItem2 = $this->complaintHelper->createComplaintItemData(array_shift($orderItems1), 'Broken!', 1, [$uploadedFile3]);
```

Comment:
nitpick, should be named `$complaintItemXY(Data)`

Please check both the files, all the variables that are created as a result of `createComplaintItemData` should be renamed properly :pray: